> feedeliza-plus@1.0.0 start
> node backend/server.js
Redis Client Connected
âœ… ConexÃ£o com banco de dados estabelecida
ðŸš€ Servidor rodando na porta 5000
DEBUG: /iam/tree - userId: 0e11f6e3-36f3-4719-9ed3-c503d4e42b9f restaurantId: fe1d1bec-5a7f-4486-ba38-81c22e131269
DEBUG: Before bump - Restaurant fe1d1bec-5a7f-4486-ba38-81c22e131269 perm_version: undefined
DEBUG: After bump - Restaurant fe1d1bec-5a7f-4486-ba38-81c22e131269 perm_version: 1
Error saving audit log: Error
    at Query.run (/app/node_modules/sequelize/lib/dialects/postgres/query.js:50:25)
    at /app/node_modules/sequelize/lib/sequelize.js:315:28
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async PostgresQueryInterface.insert (/app/node_modules/sequelize/lib/dialects/abstract/query-interface.js:308:21)
    at async AuditLog.save (/app/node_modules/sequelize/lib/model.js:2490:35)
    at async AuditLog.create (/app/node_modules/sequelize/lib/model.js:1362:12)
    at async AuditService.log (/app/backend/src/services/auditService.js:9:7)
    at async /app/backend/src/api/routes/iam.js:854:5 {
  name: 'SequelizeDatabaseError',
  parent: error: column "updated_at" of relation "audit_logs" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:285:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:122:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:35:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:524:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
      at Readable.push (node:internal/streams/readable:392:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:191:23) {
    length: 134,
    severity: 'ERROR',
    code: '42703',
    detail: undefined,
    hint: undefined,
    position: '107',
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'parse_target.c',
    line: '1075',
    routine: 'checkInsertTargets',
    sql: 'INSERT INTO "audit_logs" ("id","actor_user_id","restaurant_id","action","resource","payload","created_at","updated_at") VALUES (DEFAULT,$1,$2,$3,$4,$5,$6,$7) RETURNING "id","actor_user_id","restaurant_id","action","resource","payload","created_at","updated_at","actor_user_id","restaurant_id";',
    parameters: [
      '0e11f6e3-36f3-4719-9ed3-c503d4e42b9f',
      'fe1d1bec-5a7f-4486-ba38-81c22e131269',
      'ENTITLEMENTS_BULK_SET',
      'Restaurant:fe1d1bec-5a7f-4486-ba38-81c22e131269',
      '{"count":166}',
      '2025-09-07 13:14:23.673 +00:00',
      '2025-09-07 13:14:23.673 +00:00'
    ]
  },
  original: error: column "updated_at" of relation "audit_logs" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:285:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:122:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:35:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:524:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
      at Readable.push (node:internal/streams/readable:392:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:191:23) {
    length: 134,
    severity: 'ERROR',
    code: '42703',
    detail: undefined,
    hint: undefined,
    position: '107',
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'parse_target.c',
    line: '1075',
    routine: 'checkInsertTargets',
    sql: 'INSERT INTO "audit_logs" ("id","actor_user_id","restaurant_id","action","resource","payload","created_at","updated_at") VALUES (DEFAULT,$1,$2,$3,$4,$5,$6,$7) RETURNING "id","actor_user_id","restaurant_id","action","resource","payload","created_at","updated_at","actor_user_id","restaurant_id";',
    parameters: [
      '0e11f6e3-36f3-4719-9ed3-c503d4e42b9f',
      'fe1d1bec-5a7f-4486-ba38-81c22e131269',
      'ENTITLEMENTS_BULK_SET',
      'Restaurant:fe1d1bec-5a7f-4486-ba38-81c22e131269',
      '{"count":166}',
      '2025-09-07 13:14:23.673 +00:00',
      '2025-09-07 13:14:23.673 +00:00'
    ]
  },
  sql: 'INSERT INTO "audit_logs" ("id","actor_user_id","restaurant_id","action","resource","payload","created_at","updated_at") VALUES (DEFAULT,$1,$2,$3,$4,$5,$6,$7) RETURNING "id","actor_user_id","restaurant_id","action","resource","payload","created_at","updated_at","actor_user_id","restaurant_id";',
  parameters: [
    '0e11f6e3-36f3-4719-9ed3-c503d4e42b9f',
    'fe1d1bec-5a7f-4486-ba38-81c22e131269',
    'ENTITLEMENTS_BULK_SET',
    'Restaurant:fe1d1bec-5a7f-4486-ba38-81c22e131269',
    '{"count":166}',
    '2025-09-07 13:14:23.673 +00:00',
    '2025-09-07 13:14:23.673 +00:00'
  ]
}
DEBUG: /iam/tree - userId: 0e11f6e3-36f3-4719-9ed3-c503d4e42b9f restaurantId: fe1d1bec-5a7f-4486-ba38-81c22e131269
DEBUG: /iam/tree - userId: 0e11f6e3-36f3-4719-9ed3-c503d4e42b9f restaurantId: fe1d1bec-5a7f-4486-ba38-81c22e131269
DEBUG: Before bump - Restaurant fe1d1bec-5a7f-4486-ba38-81c22e131269 perm_version: undefined
DEBUG: After bump - Restaurant fe1d1bec-5a7f-4486-ba38-81c22e131269 perm_version: 1
Error saving audit log: Error
    at Query.run (/app/node_modules/sequelize/lib/dialects/postgres/query.js:50:25)
    at /app/node_modules/sequelize/lib/sequelize.js:315:28
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async PostgresQueryInterface.insert (/app/node_modules/sequelize/lib/dialects/abstract/query-interface.js:308:21)
    at async AuditLog.save (/app/node_modules/sequelize/lib/model.js:2490:35)
    at async AuditLog.create (/app/node_modules/sequelize/lib/model.js:1362:12)
    at async AuditService.log (/app/backend/src/services/auditService.js:9:7)
    at async /app/backend/src/api/routes/iam.js:854:5 {
  name: 'SequelizeDatabaseError',
  parent: error: column "updated_at" of relation "audit_logs" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:285:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:122:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:35:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:524:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
      at Readable.push (node:internal/streams/readable:392:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:191:23) {
    length: 134,
    severity: 'ERROR',
    code: '42703',
    detail: undefined,
    hint: undefined,
    position: '107',
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'parse_target.c',
    line: '1075',
    routine: 'checkInsertTargets',
    sql: 'INSERT INTO "audit_logs" ("id","actor_user_id","restaurant_id","action","resource","payload","created_at","updated_at") VALUES (DEFAULT,$1,$2,$3,$4,$5,$6,$7) RETURNING "id","actor_user_id","restaurant_id","action","resource","payload","created_at","updated_at","actor_user_id","restaurant_id";',
    parameters: [
      '0e11f6e3-36f3-4719-9ed3-c503d4e42b9f',
      'fe1d1bec-5a7f-4486-ba38-81c22e131269',
      'ENTITLEMENTS_BULK_SET',
      'Restaurant:fe1d1bec-5a7f-4486-ba38-81c22e131269',
      '{"count":166}',
      '2025-09-07 13:14:55.373 +00:00',
      '2025-09-07 13:14:55.373 +00:00'
    ]
  },
  original: error: column "updated_at" of relation "audit_logs" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:285:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:122:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:35:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:524:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
      at Readable.push (node:internal/streams/readable:392:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:191:23) {
    length: 134,
    severity: 'ERROR',
    code: '42703',
    detail: undefined,
    hint: undefined,
    position: '107',
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'parse_target.c',
    line: '1075',
    routine: 'checkInsertTargets',
    sql: 'INSERT INTO "audit_logs" ("id","actor_user_id","restaurant_id","action","resource","payload","created_at","updated_at") VALUES (DEFAULT,$1,$2,$3,$4,$5,$6,$7) RETURNING "id","actor_user_id","restaurant_id","action","resource","payload","created_at","updated_at","actor_user_id","restaurant_id";',
    parameters: [
      '0e11f6e3-36f3-4719-9ed3-c503d4e42b9f',
      'fe1d1bec-5a7f-4486-ba38-81c22e131269',
      'ENTITLEMENTS_BULK_SET',
      'Restaurant:fe1d1bec-5a7f-4486-ba38-81c22e131269',
      '{"count":166}',
      '2025-09-07 13:14:55.373 +00:00',
      '2025-09-07 13:14:55.373 +00:00'
    ]
  },
  sql: 'INSERT INTO "audit_logs" ("id","actor_user_id","restaurant_id","action","resource","payload","created_at","updated_at") VALUES (DEFAULT,$1,$2,$3,$4,$5,$6,$7) RETURNING "id","actor_user_id","restaurant_id","action","resource","payload","created_at","updated_at","actor_user_id","restaurant_id";',
  parameters: [
    '0e11f6e3-36f3-4719-9ed3-c503d4e42b9f',
    'fe1d1bec-5a7f-4486-ba38-81c22e131269',
    'ENTITLEMENTS_BULK_SET',
    'Restaurant:fe1d1bec-5a7f-4486-ba38-81c22e131269',
    '{"count":166}',
    '2025-09-07 13:14:55.373 +00:00',
    '2025-09-07 13:14:55.373 +00:00'
  ]
}
DEBUG: /iam/tree - userId: 0e11f6e3-36f3-4719-9ed3-c503d4e42b9f restaurantId: fe1d1bec-5a7f-4486-ba38-81c22e131269
DEBUG: Before bump - Restaurant fe1d1bec-5a7f-4486-ba38-81c22e131269 perm_version: undefined
DEBUG: After bump - Restaurant fe1d1bec-5a7f-4486-ba38-81c22e131269 perm_version: 1
Error saving audit log: Error
    at Query.run (/app/node_modules/sequelize/lib/dialects/postgres/query.js:50:25)
    at /app/node_modules/sequelize/lib/sequelize.js:315:28
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async PostgresQueryInterface.insert (/app/node_modules/sequelize/lib/dialects/abstract/query-interface.js:308:21)
    at async AuditLog.save (/app/node_modules/sequelize/lib/model.js:2490:35)
    at async AuditLog.create (/app/node_modules/sequelize/lib/model.js:1362:12)
    at async AuditService.log (/app/backend/src/services/auditService.js:9:7)
    at async /app/backend/src/api/routes/iam.js:854:5 {
  name: 'SequelizeDatabaseError',
  parent: error: column "updated_at" of relation "audit_logs" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:285:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:122:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:35:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:524:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
      at Readable.push (node:internal/streams/readable:392:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:191:23) {
    length: 134,
    severity: 'ERROR',
    code: '42703',
    detail: undefined,
    hint: undefined,
    position: '107',
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'parse_target.c',
    line: '1075',
    routine: 'checkInsertTargets',
    sql: 'INSERT INTO "audit_logs" ("id","actor_user_id","restaurant_id","action","resource","payload","created_at","updated_at") VALUES (DEFAULT,$1,$2,$3,$4,$5,$6,$7) RETURNING "id","actor_user_id","restaurant_id","action","resource","payload","created_at","updated_at","actor_user_id","restaurant_id";',
    parameters: [
      '0e11f6e3-36f3-4719-9ed3-c503d4e42b9f',
      'fe1d1bec-5a7f-4486-ba38-81c22e131269',
      'ENTITLEMENTS_BULK_SET',
      'Restaurant:fe1d1bec-5a7f-4486-ba38-81c22e131269',
      '{"count":166}',
      '2025-09-07 13:15:03.729 +00:00',
      '2025-09-07 13:15:03.729 +00:00'
    ]
  },
  original: error: column "updated_at" of relation "audit_logs" does not exist
      at Parser.parseErrorMessage (/app/node_modules/pg-protocol/dist/parser.js:285:98)
      at Parser.handlePacket (/app/node_modules/pg-protocol/dist/parser.js:122:29)
      at Parser.parse (/app/node_modules/pg-protocol/dist/parser.js:35:38)
      at Socket.<anonymous> (/app/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:524:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
      at Readable.push (node:internal/streams/readable:392:5)
      at TCP.onStreamRead (node:internal/stream_base_commons:191:23) {
    length: 134,
    severity: 'ERROR',
    code: '42703',
    detail: undefined,
    hint: undefined,
    position: '107',
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'parse_target.c',
    line: '1075',
    routine: 'checkInsertTargets',
    sql: 'INSERT INTO "audit_logs" ("id","actor_user_id","restaurant_id","action","resource","payload","created_at","updated_at") VALUES (DEFAULT,$1,$2,$3,$4,$5,$6,$7) RETURNING "id","actor_user_id","restaurant_id","action","resource","payload","created_at","updated_at","actor_user_id","restaurant_id";',
    parameters: [
      '0e11f6e3-36f3-4719-9ed3-c503d4e42b9f',
      'fe1d1bec-5a7f-4486-ba38-81c22e131269',
      'ENTITLEMENTS_BULK_SET',
      'Restaurant:fe1d1bec-5a7f-4486-ba38-81c22e131269',
      '{"count":166}',
      '2025-09-07 13:15:03.729 +00:00',
      '2025-09-07 13:15:03.729 +00:00'
    ]
  },
  sql: 'INSERT INTO "audit_logs" ("id","actor_user_id","restaurant_id","action","resource","payload","created_at","updated_at") VALUES (DEFAULT,$1,$2,$3,$4,$5,$6,$7) RETURNING "id","actor_user_id","restaurant_id","action","resource","payload","created_at","updated_at","actor_user_id","restaurant_id";',
  parameters: [
    '0e11f6e3-36f3-4719-9ed3-c503d4e42b9f',
    'fe1d1bec-5a7f-4486-ba38-81c22e131269',
    'ENTITLEMENTS_BULK_SET',
    'Restaurant:fe1d1bec-5a7f-4486-ba38-81c22e131269',
    '{"count":166}',
    '2025-09-07 13:15:03.729 +00:00',
    '2025-09-07 13:15:03.729 +00:00'
  ]
}
DEBUG: /iam/tree - userId: 0e11f6e3-36f3-4719-9ed3-c503d4e42b9f restaurantId: fe1d1bec-5a7f-4486-ba38-81c22e131269